src/components/block-css/index.js

```diff
@@ -292,9 +292,9 @@ const BlockCss = props => {
 	// TODO: why do we have this condition for the collapsedSelector, but they just do the same prepending??
 	if ( hasCollapsed ) {
 		if ( blockState === 'collapsed' ) {
```

Comment:
It also goes in here when you click on the inner column when the accordion is open @bfintal

Reply:
I think that's expected because this is called during style generation, the collapsed styles are always generated regardless whether the accordion is collapsed or not